Source/Core/AudioCommon/Mixer.cpp

```diff
@@ -88,6 +87,14 @@ unsigned int Mixer::MixerFifo::Mix(short* samples, unsigned int numSamples,
                         (static_cast<u64>(m_input_sample_rate_divisor) * 1000);
     low_watermark = std::min(low_watermark, MAX_SAMPLES / 2);
 
+    // Throw away extra audio when the writer exits fast forward.
+    if (should_resync && numLeft > low_watermark)
```

Comment:
I think this branch should be skipped if auto stretching is eanbled? We never really want to throw away samples in that case, we prioritize playing all of them, even if they play at a different "speed".

EDIT: nevermind, this is already skipped is audio stretching is enabled.